docs/security-model.mdx

```diff
@@ -3,30 +3,35 @@ id: security-model
 title: Security Model
 ---
 
-The security model used by Ory is designed to deliver the most secure and hassle-free solution to all users.
-
-To achieve that, Ory manages:
+Orys' security model delivers the most secure and hassle-free solution to users by managing:
```

Comment:
```suggestion
The security model used by Ory delivers the most secure and hassle-free solution to users by managing:
```

`Orys'` is a possessive form of a plural noun, which is not correct. Let's stick to the previous approach, k?